lib/services/schedule.js

```diff
@@ -13,26 +14,34 @@ class GoogleCal {
 
   findNextStreamDay() {
     return this.getListOfUpcomingEvents(this.configuration.GOOGLE_CALENDAR_ID).then((events) => {
-      const now = moment.tz('UTC');
       const nextEvent = events.find((event) => {
```

Comment:
You're just trying to get the next event and return `null` if there isn't one? Why not immediately return `null` if `events` is empty? I don't think the response will ever contain an event without a start, so you're already doing this, but in a convoluted way.